ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/LagReportingAgentTest.java

```diff
@@ -214,27 +214,37 @@ public void shouldReceiveLags_listAllCurrentPositions() {
 
     // Then:
     ImmutableMap<KsqlHostInfoEntity, HostStoreLags> allLags = lagReportingAgent.getAllLags();
-    LagInfoEntity lag = allLags.get(HOST_ENTITY1).getStateStoreLags(QUERY_STORE_A)
+    LagInfoEntity lag = Optional.ofNullable(allLags.get(HOST_ENTITY1))
```

Comment:
Can this happen or is it just to make spotbugs happy? same for the others

Reply:
Most of the changes are there to make spotbugs happy because guava added some NonNull tags to its data structures. For the cases, I verified a null could theoretically happen.